### ISSUE TITLE
If remapping MERRA2, either require 21z as hour or don't ask?

### ISSUE THREAD

**Issue by mathomp4** [GEOS-ESM/GEOS_Util#25]
When running `remap_restarts.py` from the command line and you ask to regrid from MERRA-2, it is possible to specify an impossible time:

```
$ /discover/swdev/mathomp4/Models/GEOSgcm-FixMAPL2081/GEOSgcm/install-Release/bin/remap_restarts.py
? Would you like to remap archived MERRA-2 restarts? Yes
? From what restart date/time would you like to remap? (must be 10 digits: yyyymmddhh) 2000041406
? Enter the directory for new restarts:
```

There are no 06z restarts from MERRA-2, only 21z.

So this leads to two ways of thinking:

1. If MERRA-2, you only ask for date, not for the time. Internally, we append the `21` and the yaml file gets that, but we don't ask the user
2. If MERRA-2, we ask for the time but die or cycle if a non-21 time is supplied

Myself or @weiyuan-jiang could probably code this, but I'll ping @tclune and @JulesKouatchou for more thoughts.

I think the first is more elegant, but it does mean we need two different questions for a date depending on the first question.

**Comment by tclune** [comment]
I prefer a modified version of option 1:   With MERRA-2, only ask for a date.  But then _report_ to the user that it will be for 21z.  I.e, make it explicit so that there is no surprise.   Don't know if Questionary lets you do that, but presumably you can have a "Null Question"?

**Comment by weiyuan-jiang** [comment]
Option 1 is taken and pushed to my new branch #19

**Comment by biljanaorescanin** [comment]
This issue is addressed with #19
